.vib/grafana-mimir/cypress/cypress/integration/grafana_mimir_spec.js

```diff
@@ -0,0 +1,41 @@
+/// <reference types="cypress" />
+import body from '../fixtures/metric-body.json';
+import {
+  lastMinuteTimestamp,
+  random
+} from '../support/utils';
+
+it('can push metrics', () => {
+  // Using Open Telemetry endpoint because it allow us to use plain connection.
+  // Regular prometheus endpoint requires snappy compression.
+  const upToDateJson = JSON.stringify(body).replace(
+    '"timestamp_placeholder"',
+    lastMinuteTimestamp
+  ).replace(
+    '"integer_placeholder"',
+    random
+  );
+  cy.request({
+    method: 'POST',
+    headers: { 'X-Scope-OrgID': 'demo' },
```

Comment:
As you are reusing this value (`demo`) later on, I'd use a constant to store it.